### ISSUE TITLE
Cluster hover event

### ISSUE THREAD

**Issue by mvidalgarcia** [yuzhva/react-leaflet-markercluster#67]
I couldn't find the `clustermouseover` event that exists in https://github.com/Leaflet/Leaflet.markercluster
It would be nice to have some hover event in the marker cluster, it'd useful e.g. to print a tooltip. For instance something like this:
```jsx
<MarkerClusterGroup onClusterMouseover={() => console.log('hover!')}>
...
</MarkerClusterGroup>
```

**Comment by jbccollins** [comment]
Given how the Leaflet.markercluster library handles these events generically I'd like to see it done in the same way. https://github.com/Leaflet/Leaflet.markercluster#events

Any valid event for markers should be able to be prefixed by 'cluster', not just mouseover.

**Comment by jbccollins** [comment]
This worked for me. I can't make a PR just now but if someone wants to use this as inspiration then go for it. Probably some simplifications on the code duplication are possible if you want to do that.

```javascript
class CustomMarkerClusterGroup extends MarkerClusterGroup {
  initEventListeners(markerClusterGroup) {
    const ON_CLUSTER = 'onCluster';
    const ON_MARKER = 'onMarker';
    const ON_POPUP = 'onPopup'
    Object.keys(this.props).forEach(k => {
      if (k.indexOf(ON_CLUSTER) === 0) {
        const eventName = k.replace(ON_CLUSTER, 'cluster').toLowerCase();
        markerClusterGroup.on(eventName, (cluster) => {
          this.props[k](cluster.layer);
        });
      } else if (k.indexOf(ON_MARKER) === 0) {
        const eventName = k.replace(ON_MARKER, '').toLowerCase();
        markerClusterGroup.on(eventName, (marker) => {
          this.props[k](marker.layer);
        });
      } else if (k.indexOf(ON_POPUP) === 0) {
        const eventName = k.replace(ON_POPUP, '').toLowerCase();
        markerClusterGroup.on(eventName, (map) => {
          this.props[k](map.popup);
        });
      }
    })  
  }
}
```

**Comment by NilanshBansal** [comment]
ThankYou @jbccollins 👍

**Comment by YUzhva** [comment]
This issue was resolved in beta `v2.0.0-rc1` release.

There are available all leaflet [map Interaction events](https://leafletjs.com/reference-1.3.0.html#map-interaction-events) like dblclick/mouseup/mousedown etc.

Just v2.x of that plugin requires react-leaflet@^2.0.0.

**Comment by yuzhva** [comment]
the `v2.0.0` has been released...

```javascript
<MarkerClusterGroup onClusterMouseover={() => console.log('hover!')}>
...
</MarkerClusterGroup>
```

works just fine 👌 
Here is a link for the CodeSandbox Getting Started:
https://codesandbox.io/s/react-leaflet-markercluster-getting-started-9binx
